pages/common/g++.md

```diff
@@ -0,0 +1,19 @@
+# G Plus Plus
+
+> G++ is a compiler for C++. Member of GNU GCC.
```

Comment:
How does this sound -
```
> Compiles C++ source files.
> Part of GCC (GNU Compiler Collection).
```

Reply:
Rest of the changes make sense, will make those soon.